source/Subsystem/Yellowstone/Vertex.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#define GLM_ENABLE_EXPERIMENTAL
+#include <glm/gtx/hash.hpp>
+#include <glm/vec3.hpp>
+#include <glm/vec2.hpp>
+#include <assimp/vector2.h>
+#include <assimp/vector3.h>
+
+struct Vertex
+{
+	glm::vec3 myPosition;
+	glm::vec2 myTextureCoordinates;
+
+	bool operator==(const Vertex& aOther) const
+	{
+		return myPosition == aOther.myPosition &&myTextureCoordinates == aOther.myTextureCoordinates;
+	}
+};
+
+inline void CombineHash(std::size_t& aSeed) {}
+
+template <typename T, typename... Rest>
+inline void CombineHash(std::size_t& aSeed, const T& aValue, Rest... aRest)
+{
+	std::hash<T> hasher;
+	aSeed ^= hasher(aValue) + 0x9e3779b9 + (aSeed << 6) + (aSeed >> 2);
```

Comment:
I have no idea what's happening here.

Reply:
It's based on boost:make_unique. There are betters ways to make it really unique using some C++17 magic, but this is fine for now.